recipes/aws-c-cal/all/conanfile.py

```diff
@@ -35,6 +34,11 @@ def _source_subfolder(self):
     def _needs_openssl(self):
         return self.settings.os != "Windows" and not tools.is_apple_os(self.settings.os)
 
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
```

Comment:
Why was this changed? Does it save a few patches from being copied?

Reply:
Yes, I have done this for other recipes as well.

All versions in this recipe require the same patch now, but this may change in the future, so the change was made for the purpose of omitting copying.

Reply:
Thanks for confirming!